ui/src/pages/Login/Login.tsx

```diff
@@ -1,7 +1,26 @@
 import React from "react";
 
 export function Login() {
-  return(
-    <button>This is my button</button>
-  )
-}
\ No newline at end of file
+  return (
+    <div className="login">
+      <>
+        <img src="/images/aaaa-scream.gif" className="login__scream" />
+        <input className="login__input" placeholder="Email" />
+      </>
+      <>
+        <img src="/images/terraria-bunny1.gif" className="login__bunnies" />
+        <input className="login__input" placeholder="Username" />
```

Comment:
I feel like username should come before password.  Let me know if you think otherwise, but it just makes sense to me